sync/sync.go

```diff
@@ -234,13 +254,10 @@ Register:
 		}
 		return nil
 	}
-	logrus.Infof("CheckAndResetFailedRebuild %v", replicaAddress)
-	if err := t.checkAndResetFailedRebuild(replicaAddress, s); err != nil {
-		return fmt.Errorf("CheckAndResetFailedRebuild failed, error: %s", err.Error())
-	}
-
 	logrus.Infof("Adding replica %s in WO mode", replicaAddress)
+	s.SetPreload(false)
```

Comment:
some comment here by saying 'CreateReplica goes to controller and it triggers Open call with replica.. but we don't want to perform preload in this case.. so setting Preload to false

Reply:
done